server/src/main/java/org/elasticsearch/search/fetch/FetchPhaseExecutionException.java

```diff
@@ -20,10 +20,6 @@ public FetchPhaseExecutionException(SearchShardTarget shardTarget, String msg, T
         super(shardTarget, "Fetch Failed [" + msg + "]", t);
     }
 
-    public FetchPhaseExecutionException(SearchShardTarget shardTarget, String msg) {
-        super(shardTarget, "Fetch Failed [" + msg + "]");
-    }
```

Comment:
Shall we assert that the cause is never null in these exceptions?

Reply:
OK, added.